process.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+with open(sys.argv[1], 'r') as f: 
```

Comment:
This isn't critical, but for bonus points, you can avoid needing to take a file name as input by just reading from standard input… that is, read directly from `sys.stdin`. Then you would invoke this script like:

```
python3 process.py < input.txt > output.txt
```

to read data from `input.txt` and write it to `output.txt`.